Polygon/polygon-lens/src/mappings/mappingHandlers.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  PostCreatedLog,
+  ProfileCreatedLog,
+  FollowedLog,
+} from "../types/abi-interfaces/LensHubAbi";
+import { Account, Post, Profile, Follow } from "../types";
+import assert from "assert";
+import { BigNumber } from "ethers";
+
+export async function getOrCreateAccount(
+  accountAddress: string
+): Promise<Account> {
+  let account = await Account.get(accountAddress);
+  if (account == null) {
+    account = Account.create({
+      id: accountAddress,
+    });
+  }
+  return account;
+}
+
+export async function getOrCreateFollow(
+  accountAddress: string
+): Promise<Follow> {
+  let follow = await Follow.get(accountAddress);
+  if (follow == null) {
+    follow = Follow.create({ id: accountAddress });
+  }
+  return follow;
+}
+
+export function getNewPublicactionId(profileId: BigInt, pubId: BigInt): string {
+  return profileId.toString().concat("-").concat(pubId.toString());
+}
+
+export async function getOrCreatePost(pubId: BigInt): Promise<Post> {
+  let post = await Post.get(pubId.toString());
+  if (post === undefined) {
```

Comment:
You use null checks above, and undefined checks here. Even though they both work, its confusing why you switch between them
```suggestion
  if (!post) {
```